spec/features/edit_page_spec.rb

```diff
@@ -65,13 +65,23 @@
     new_email = "example@example.com"
     customer = create(:customer)
 
-    visit edit_admin_customer_path(customer)
-    fill_in "Name", with: new_name
-    fill_in "Email", with: new_email
-    click_on "Update Customer"
+    translations = {
+      activerecord: {
+        models: {
+          customer: "Custom"
```

Comment:
What do you think of naming the overridden name something like "Custom name"? At first glance, I thought this was a mistake in translations, rather than it being set by you

Reply:
Sure